easybuild/easyconfigs/b/byacc/byacc-2.0.20240109-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = 'byacc'
+version = '2.0.20240109'
+
+homepage = 'http://invisible-island.net/byacc/byacc.html'
+description = """Berkeley Yacc (byacc) is generally conceded to be the best yacc variant available.
+ In contrast to bison, it is written to avoid dependencies upon a particular compiler.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+source_urls = ['https://invisible-island.net/archives/byacc/current/']
+sources = ['byacc.tar.gz']
```

Comment:
This could create problems when downloading a different version of `byacc.tar.gz`. I made a suggestion in https://github.com/easybuilders/easybuild-easyconfigs/pull/21344